packages/lib/src/Button.wc.svelte

```diff
@@ -22,27 +16,28 @@
     lg: "px-6 py-3 text-lg",
   };
 
-  const variantClasses = {
-    primary: {
-      solid:
-        "bg-[var(--sesamy-button-bg)] text-[var(--sesamy-button-text)] hover:bg-[var(--sesamy-button-bg-hover)] focus:ring-[var(--sesamy-button-ring)]",
-      outline:
-        "bg-transparent border border-[var(--sesamy-button-border)] text-[var(--sesamy-button-text)] hover:bg-[var(--sesamy-button-bg-hover)] hover:text-[var(--sesamy-button-text-hover)] focus:ring-[var(--sesamy-button-ring)]",
-    },
-    secondary: {
-      solid:
-        "bg-[var(--sesamy-button-secondary-bg)] text-[var(--sesamy-button-secondary-text)] hover:bg-[var(--sesamy-button-secondary-bg-hover)] focus:ring-[var(--sesamy-button-secondary-ring)]",
-      outline:
-        "bg-transparent border border-[var(--sesamy-button-secondary-border)] text-[var(--sesamy-button-secondary-text)] hover:bg-[var(--sesamy-button-secondary-bg-hover)] hover:text-[var(--sesamy-button-secondary-text-hover)] focus:ring-[var(--sesamy-button-secondary-ring)]",
-    },
+  const getButtonClasses = (variant: Variant) => {
+    switch (variant) {
+      case "primary":
+        return "bg-blue-500 text-white hover:bg-blue-600 focus:ring-blue-300";
+      case "secondary":
+        return "bg-transparent border border-blue-500 text-blue-500 hover:bg-blue-50 hover:text-blue-600 focus:ring-blue-300";
+      case "tertiary":
+        return "text-blue-500 hover:text-blue-600 focus:ring-blue-300";
+    }
   };
 
-  $: classes = twMerge(
-    baseClasses,
-    sizeClasses[size],
-    variantClasses[variant][outline ? "outline" : "solid"],
-    disabled && "opacity-50 cursor-not-allowed",
-  );
+  // Define reactive state for classes
+  let classes = $state("");
+
+  $effect(() => {
```

Comment:
So, this is needed to make it react to changes in the attributes. The $: classes we used before isn't allowed when using the $Props above.

Reply:
My previous question asked about exactly this. Let's look at it on monday.

If `$props()` are passed down as `$state()` they should react to changes without this black magic :)

Reply:
So.. It might be possible to just change the $props to $state? Will they both be populated by the attributes of the component?

Reply:
Tested a bit without getting it to work. Let's check together next week.